unitTest/@testFEMComputationalFluidDynamicsAnalysis/testFEM4NavierStokesSteadyStateFlowAroundCylinder2D.m

```diff
@@ -98,19 +99,18 @@ function testFEM4NavierStokesSteadyStateFlowAroundCylinder2D(testCase)
     gaussInt,caseName,'');
 
 %% 7. Calculate drag and lift force from the nodal forces
-Fx = calculateDragForce (FComplete, fldMsh, homDBC, parameters);
-Fy = calculateLiftForce (FComplete, fldMsh, homDBC, parameters);
+[Fx, Fy, ~] = calculateForce(FComplete, fldMsh, homDBC, parameters, 2);
```

Comment:
Now that the functions are a little cleaner it is clear that one desides upon which is the object over which the quantities of interest are computed by excluding all homDBC DOFs which are not located at the bounday of a rectangle. **This is not the way to go as it clearly problem specific and error prone**. Therefore, we will have to update the GiD problem type so that the user can select the boundary/-ies which are supposed to represent the body over which the quantity/-ies of interest is/are to be computed and assign to them a name. Then, the parser in Matlab needs to be extended to read this extra block from the generated GiD input file and compute the quantity of interest that the user specifies over each different boundary. Is it clear what I mean?

Reply:
The GiD problem type for cane can be found under,

C:\FiniteElementAnalysisProgramStructuralAnalysisInstituteTUM\cane\problemTypeGiD

you can obtain it in your branch by merging the master as follows,

git merge master

there should be no conflicts as no other change is done in the master.

Cheers!

Reply:
I am sorry the above directory is the one in my computer, yours should be something like,

~\cane\problemTypeGiD